src/load/azext_load/data_plane/utils/utils.py

```diff
@@ -821,3 +832,62 @@ def validate_engine_data_with_regionwiseload_data(engine_instances, regionwise_e
             f"Sum of engine instances in regionwise load test configuration ({total_engines}) "
             f"should be equal to total engine instances ({engine_instances})"
         )
+
+
+def _get_metrics_from_sampler(test_run, sampler_name, metric_name):
+    return test_run.get("testRunStatistics", {}).get(sampler_name, {}).get(metric_name)
+
+
+def generate_trends_row(test_run, response_time_aggregate=None):
+    trends = {
+        LoadTestTrendsKeys.NAME: test_run.get("testRunId"),
```

Comment:
portal doesn't have display name, it uses test run id in the Trends tab? Moreover, in cli, it's the test run id which I believe is a better identifier since that is what the user sets at the time of creation. @saloniagrawal1997 , can you please verify the header keys to be used for the compare-to-baseline cmd?

Reply:
As seen on the portal is good, Test Run Name is autogenerated always. Neither Test RUN ID nor name is specified ny the user. Lets keep the table same as portal.

Reply:
Okay with keeping it as same as the portal.

Reply:
> As seen on the portal is good, Test Run Name is autogenerated always. Neither Test RUN ID nor name is specified ny the user. Lets keep the table same as portal.

@saloniagrawal1997 , seems there's no field which specifies name for a test run, can you please confirm what is to be treated as Name? https://learn.microsoft.com/en-us/rest/api/loadtesting/dataplane/load-test-run/get-test-run?view=rest-loadtesting-dataplane-2024-07-01-preview&tabs=HTTP

Reply:
using test run display name for Name